airflow/providers/google/cloud/hooks/dataflow.py

```diff
@@ -147,9 +147,16 @@ class _DataflowJobsController(LoggingMixin):
         not by specific job ID, then actions will be performed on all matching jobs.
     :param drain_pipeline: Optional, set to True if want to stop streaming job by draining it
         instead of canceling.
+    :param wait_until_finished: If True, wait for the end of pipeline execution before exiting. If False,
+        it only waits for it to starts (``JOB_STATE_RUNNING``).
+
+        The default behavior depends on the type of pipeline:
```

Comment:
If False,
        it only waits for it to starts (``JOB_STATE_RUNNING``) ->

This might wait a long time (hours+). A job might be created and scheduled to run later (e.g. flex resource scheduling).

Suggestion: if False, submit the job and return without waiting

Reply:
@aaltay I changed behavior for `wait_until_finished=False`.
Now it will not wait for job to be in running state. It could be checked by the sensor.